Public send emails

tcitworld opened the pull request:
Just to have a preview for https://github.com/owncloud/calendar/pull/605

codecrusher64 commented:
I checked it on my platform with Firefox and Chrome. Both show share icon correctly now.

Aschen commented:
Ok with Firefox 34.0.5 on Windows 7.

Thanks

jbtbnl commented:
@georgehrke should this be backported?

georgehrke commented:
yes, I'll take care

jancborchardt commented:
Wait – icon-shared is a different icon than icon-share! And one we shouldn’t use anymore.

jancborchardt commented:
We need to fix this in AdBlock and let the Adblock devs know to exclude that.

RealRancor commented:
Yes, as core is also affected by this Adblock rule:

https://github.com/owncloud/core/issues/8360#issuecomment-69195136

tcitworld committed:
Start work to publish a calendar

more interface changes

Working interface

Put correct URL

ask for urls

updates for server changes

Finish publish feature

Clean things up

Rebase branch after CalendarModel refactor

Small property fix

try go public

work

Make everything work

Remove un-necessary call

Fix rebase issue and don't show ourselves inside the sharees

Clean things up, show both CalDAV & Web Interface URLs and remove unused code in public controller

Fix undefined bug by moving to service

put back empty emailAddress as it's needed

Move CalDAV address to public view, make necessary changes for iframes, add file download on public view and integration code

remove unused sandbox

clean rebase

Use original controllers

Additionnal checks for pre-publish-url

Simplify public page calendar properties acquisition & add a publishable status for a calendar

fix day/week/month buttons not accessible in public mode

Don't show notification if we are on the public page

rebased

add basic tests

Even more tests (publish & unpublish)

And a filename typo

change share icons behaviour consistent with File app

fix for urls finishing with #

use es6 endsWith function

fix double slash issue in urls

tcitworld committed:
remove header

tcitworld committed:
try to send emails

tcitworld committed:
try to put sender in the loop

tcitworld committed:
fix domain

tcitworld committed:
bring l10n

tcitworld committed:
fix js

tcitworld committed:
put calendar name into the mail

tcitworld committed:
fix typo
